larpandoracontent/LArHelpers/LArVertexHelper.cc

```diff
@@ -58,4 +59,21 @@ LArVertexHelper::ClusterDirection LArVertexHelper::GetClusterDirectionInZ(
     throw StatusCodeException(STATUS_CODE_FAILURE);
 }
 
+//-----------------------------------------------------------------------------------------------------------------------------------------
+
+bool LArVertexHelper::IsInFiducialVolume(const CartesianVector &vertex, const std::string &detector)
+{
+    if (detector == "dune_fd_hd")
+    {
+        const float x{vertex.GetX()};
+        const float y{vertex.GetY()};
+        const float z{vertex.GetZ()};
+        return -310 < x && x < 310 && -550 < y && y < 550 && 50 < z && z < 1244;
```

Comment:
Can these be defined against the detector boundaries?  This would defend against using a larger geometry.

Also I guess the magic numbers should be floats.